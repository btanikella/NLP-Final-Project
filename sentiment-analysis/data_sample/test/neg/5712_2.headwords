soft-porn
made
excuse
's
bad
Let
succeeded
's
