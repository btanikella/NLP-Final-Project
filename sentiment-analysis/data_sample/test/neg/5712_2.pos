DT VBZ JJ NN : DT JJ NN NN VBG IN CD NN : NN
PRP RB VBD NN JJ  JJ TO VB  IN PRP VBP PRP TO VB DT RB CC VB PRP WP DT NN VBD
WP DT VBZ  VBZ DT NN IN NNP NNP TO VB RP PRP$ JJ NN CC IN NNP NNP TO VB DT NN
DT VBZ PRP
NNP POS NN VBZ RB JJ PRP VBZ JJ CC NNP  IN DT JJ NN  VBZ RB JJR IN NNS TO DT NNP IN DT JJ NN NNS CC DT JJ NN  CC DT NN VBZ RB JJ : DT NN NN CC DT NN : PRP VBZ JJ TO VB
VB PRP VB PRP : DT NN VBD VBN IN JJ NN : TO VB JJ NNS
IN DT NN  PRP RB VBD
IN PRP VBP  PRP VBZ WRB PRP VBD PRP DT NN VBG DT NN IN NNP POS NNS  CC PRP RB VBD DT NN  NN VBG TO VB DT RP IN  JJ  CC NN JJR IN JJ NN VBP RB VBG PRP
