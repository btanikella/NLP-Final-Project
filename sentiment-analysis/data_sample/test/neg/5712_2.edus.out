Sentence #1 (16 tokens):
This is high-gloss soft-porn; a boring soap opera concentrating on one thing: sex.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=is CharacterOffsetBegin=9 CharacterOffsetEnd=11 PartOfSpeech=VBZ] [Text=high-gloss CharacterOffsetBegin=12 CharacterOffsetEnd=22 PartOfSpeech=JJ] [Text=soft-porn CharacterOffsetBegin=23 CharacterOffsetEnd=32 PartOfSpeech=NN] [Text=; CharacterOffsetBegin=32 CharacterOffsetEnd=33 PartOfSpeech=:] [Text=a CharacterOffsetBegin=34 CharacterOffsetEnd=35 PartOfSpeech=DT] [Text=boring CharacterOffsetBegin=36 CharacterOffsetEnd=42 PartOfSpeech=JJ] [Text=soap CharacterOffsetBegin=43 CharacterOffsetEnd=47 PartOfSpeech=NN] [Text=opera CharacterOffsetBegin=48 CharacterOffsetEnd=53 PartOfSpeech=NN] [Text=concentrating CharacterOffsetBegin=54 CharacterOffsetEnd=67 PartOfSpeech=VBG] [Text=on CharacterOffsetBegin=68 CharacterOffsetEnd=70 PartOfSpeech=IN] [Text=one CharacterOffsetBegin=71 CharacterOffsetEnd=74 PartOfSpeech=CD] [Text=thing CharacterOffsetBegin=75 CharacterOffsetEnd=80 PartOfSpeech=NN] [Text=: CharacterOffsetBegin=80 CharacterOffsetEnd=81 PartOfSpeech=:] [Text=sex CharacterOffsetBegin=82 CharacterOffsetEnd=85 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=85 CharacterOffsetEnd=86 PartOfSpeech=.] 
root(ROOT-0, soft-porn-4)
nsubj(soft-porn-4, This-1)
cop(soft-porn-4, is-2)
amod(soft-porn-4, high-gloss-3)
punct(soft-porn-4, ;-5)
det(opera-9, a-6)
amod(opera-9, boring-7)
nn(opera-9, soap-8)
parataxis(soft-porn-4, opera-9)
partmod(opera-9, concentrating-10)
prep(concentrating-10, on-11)
num(thing-13, one-12)
pobj(on-11, thing-13)
punct(thing-13, :-14)
dep(thing-13, sex-15)
punct(soft-porn-4, .-16)

Sentence #2 (26 tokens):
They actually made sex boring, sad to say, because I defy you to watch this casually and tell me what the storyline was.
[Text=They CharacterOffsetBegin=91 CharacterOffsetEnd=95 PartOfSpeech=PRP] [Text=actually CharacterOffsetBegin=96 CharacterOffsetEnd=104 PartOfSpeech=RB] [Text=made CharacterOffsetBegin=105 CharacterOffsetEnd=109 PartOfSpeech=VBD] [Text=sex CharacterOffsetBegin=110 CharacterOffsetEnd=113 PartOfSpeech=NN] [Text=boring CharacterOffsetBegin=114 CharacterOffsetEnd=120 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=120 CharacterOffsetEnd=121 PartOfSpeech=,] [Text=sad CharacterOffsetBegin=122 CharacterOffsetEnd=125 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=126 CharacterOffsetEnd=128 PartOfSpeech=TO] [Text=say CharacterOffsetBegin=129 CharacterOffsetEnd=132 PartOfSpeech=VB] [Text=, CharacterOffsetBegin=132 CharacterOffsetEnd=133 PartOfSpeech=,] [Text=because CharacterOffsetBegin=134 CharacterOffsetEnd=141 PartOfSpeech=IN] [Text=I CharacterOffsetBegin=142 CharacterOffsetEnd=143 PartOfSpeech=PRP] [Text=defy CharacterOffsetBegin=144 CharacterOffsetEnd=148 PartOfSpeech=VBP] [Text=you CharacterOffsetBegin=149 CharacterOffsetEnd=152 PartOfSpeech=PRP] [Text=to CharacterOffsetBegin=153 CharacterOffsetEnd=155 PartOfSpeech=TO] [Text=watch CharacterOffsetBegin=156 CharacterOffsetEnd=161 PartOfSpeech=VB] [Text=this CharacterOffsetBegin=162 CharacterOffsetEnd=166 PartOfSpeech=DT] [Text=casually CharacterOffsetBegin=167 CharacterOffsetEnd=175 PartOfSpeech=RB] [Text=and CharacterOffsetBegin=176 CharacterOffsetEnd=179 PartOfSpeech=CC] [Text=tell CharacterOffsetBegin=180 CharacterOffsetEnd=184 PartOfSpeech=VB] [Text=me CharacterOffsetBegin=185 CharacterOffsetEnd=187 PartOfSpeech=PRP] [Text=what CharacterOffsetBegin=188 CharacterOffsetEnd=192 PartOfSpeech=WP] [Text=the CharacterOffsetBegin=193 CharacterOffsetEnd=196 PartOfSpeech=DT] [Text=storyline CharacterOffsetBegin=197 CharacterOffsetEnd=206 PartOfSpeech=NN] [Text=was CharacterOffsetBegin=207 CharacterOffsetEnd=210 PartOfSpeech=VBD] [Text=. CharacterOffsetBegin=210 CharacterOffsetEnd=211 PartOfSpeech=.] 
root(ROOT-0, made-3)
nsubj(made-3, They-1)
advmod(made-3, actually-2)
dobj(made-3, sex-4)
amod(sex-4, boring-5)
punct(sex-4, ,-6)
amod(sex-4, sad-7)
aux(say-9, to-8)
xcomp(sad-7, say-9)
punct(made-3, ,-10)
mark(defy-13, because-11)
nsubj(defy-13, I-12)
advcl(made-3, defy-13)
dobj(defy-13, you-14)
aux(watch-16, to-15)
xcomp(defy-13, watch-16)
det(casually-18, this-17)
dobj(watch-16, casually-18)
cc(watch-16, and-19)
conj(watch-16, tell-20)
dobj(tell-20, me-21)
dobj(was-25, what-22)
det(storyline-24, the-23)
nsubj(was-25, storyline-24)
dep(tell-20, was-25)
punct(made-3, .-26)

Sentence #3 (25 tokens):
What this is, is an excuse for Kim Bassinger to show off her great body and for Mickey Rourke to smirk a lot.
[Text=What CharacterOffsetBegin=216 CharacterOffsetEnd=220 PartOfSpeech=WP] [Text=this CharacterOffsetBegin=221 CharacterOffsetEnd=225 PartOfSpeech=DT] [Text=is CharacterOffsetBegin=226 CharacterOffsetEnd=228 PartOfSpeech=VBZ] [Text=, CharacterOffsetBegin=228 CharacterOffsetEnd=229 PartOfSpeech=,] [Text=is CharacterOffsetBegin=230 CharacterOffsetEnd=232 PartOfSpeech=VBZ] [Text=an CharacterOffsetBegin=233 CharacterOffsetEnd=235 PartOfSpeech=DT] [Text=excuse CharacterOffsetBegin=236 CharacterOffsetEnd=242 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=243 CharacterOffsetEnd=246 PartOfSpeech=IN] [Text=Kim CharacterOffsetBegin=247 CharacterOffsetEnd=250 PartOfSpeech=NNP] [Text=Bassinger CharacterOffsetBegin=251 CharacterOffsetEnd=260 PartOfSpeech=NNP] [Text=to CharacterOffsetBegin=261 CharacterOffsetEnd=263 PartOfSpeech=TO] [Text=show CharacterOffsetBegin=264 CharacterOffsetEnd=268 PartOfSpeech=VB] [Text=off CharacterOffsetBegin=269 CharacterOffsetEnd=272 PartOfSpeech=RP] [Text=her CharacterOffsetBegin=273 CharacterOffsetEnd=276 PartOfSpeech=PRP$] [Text=great CharacterOffsetBegin=277 CharacterOffsetEnd=282 PartOfSpeech=JJ] [Text=body CharacterOffsetBegin=283 CharacterOffsetEnd=287 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=288 CharacterOffsetEnd=291 PartOfSpeech=CC] [Text=for CharacterOffsetBegin=292 CharacterOffsetEnd=295 PartOfSpeech=IN] [Text=Mickey CharacterOffsetBegin=296 CharacterOffsetEnd=302 PartOfSpeech=NNP] [Text=Rourke CharacterOffsetBegin=303 CharacterOffsetEnd=309 PartOfSpeech=NNP] [Text=to CharacterOffsetBegin=310 CharacterOffsetEnd=312 PartOfSpeech=TO] [Text=smirk CharacterOffsetBegin=313 CharacterOffsetEnd=318 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=319 CharacterOffsetEnd=320 PartOfSpeech=DT] [Text=lot CharacterOffsetBegin=321 CharacterOffsetEnd=324 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=324 CharacterOffsetEnd=325 PartOfSpeech=.] 
root(ROOT-0, excuse-7)
dobj(is-3, What-1)
nsubj(is-3, this-2)
dep(excuse-7, is-3)
punct(excuse-7, ,-4)
cop(excuse-7, is-5)
det(excuse-7, an-6)
mark(show-12, for-8)
nn(Bassinger-10, Kim-9)
nsubj(show-12, Bassinger-10)
aux(show-12, to-11)
advcl(excuse-7, show-12)
prt(show-12, off-13)
poss(body-16, her-14)
amod(body-16, great-15)
dobj(show-12, body-16)
cc(show-12, and-17)
conj(show-12, for-18)
nn(Rourke-20, Mickey-19)
pobj(for-18, Rourke-20)
aux(smirk-22, to-21)
infmod(Rourke-20, smirk-22)
det(lot-24, a-23)
dobj(smirk-22, lot-24)
punct(excuse-7, .-25)

Sentence #4 (4 tokens):
That's it.
[Text=That CharacterOffsetBegin=330 CharacterOffsetEnd=334 PartOfSpeech=DT] [Text='s CharacterOffsetBegin=334 CharacterOffsetEnd=336 PartOfSpeech=VBZ] [Text=it CharacterOffsetBegin=337 CharacterOffsetEnd=339 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=339 CharacterOffsetEnd=340 PartOfSpeech=.] 
root(ROOT-0, 's-2)
nsubj('s-2, That-1)
nsubj('s-2, it-3)
punct('s-2, .-4)

Sentence #5 (55 tokens):
Rourke's smugness is so bad it's sickening and Bassinger, despite the great figure, looks cheap more than beautiful.Kudos to the photographer for some nice closeup shots and some wonderful color, but the story is so weak - no character development and no plot - it's unable to compensate.
[Text=Rourke CharacterOffsetBegin=345 CharacterOffsetEnd=351 PartOfSpeech=NNP] [Text='s CharacterOffsetBegin=351 CharacterOffsetEnd=353 PartOfSpeech=POS] [Text=smugness CharacterOffsetBegin=354 CharacterOffsetEnd=362 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=363 CharacterOffsetEnd=365 PartOfSpeech=VBZ] [Text=so CharacterOffsetBegin=366 CharacterOffsetEnd=368 PartOfSpeech=RB] [Text=bad CharacterOffsetBegin=369 CharacterOffsetEnd=372 PartOfSpeech=JJ] [Text=it CharacterOffsetBegin=373 CharacterOffsetEnd=375 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=375 CharacterOffsetEnd=377 PartOfSpeech=VBZ] [Text=sickening CharacterOffsetBegin=378 CharacterOffsetEnd=387 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=388 CharacterOffsetEnd=391 PartOfSpeech=CC] [Text=Bassinger CharacterOffsetBegin=392 CharacterOffsetEnd=401 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=401 CharacterOffsetEnd=402 PartOfSpeech=,] [Text=despite CharacterOffsetBegin=403 CharacterOffsetEnd=410 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=411 CharacterOffsetEnd=414 PartOfSpeech=DT] [Text=great CharacterOffsetBegin=415 CharacterOffsetEnd=420 PartOfSpeech=JJ] [Text=figure CharacterOffsetBegin=421 CharacterOffsetEnd=427 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=427 CharacterOffsetEnd=428 PartOfSpeech=,] [Text=looks CharacterOffsetBegin=429 CharacterOffsetEnd=434 PartOfSpeech=VBZ] [Text=cheap CharacterOffsetBegin=435 CharacterOffsetEnd=440 PartOfSpeech=RB] [Text=more CharacterOffsetBegin=441 CharacterOffsetEnd=445 PartOfSpeech=JJR] [Text=than CharacterOffsetBegin=446 CharacterOffsetEnd=450 PartOfSpeech=IN] [Text=beautiful.Kudos CharacterOffsetBegin=451 CharacterOffsetEnd=466 PartOfSpeech=NNS] [Text=to CharacterOffsetBegin=467 CharacterOffsetEnd=469 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=470 CharacterOffsetEnd=473 PartOfSpeech=DT] [Text=photographer CharacterOffsetBegin=474 CharacterOffsetEnd=486 PartOfSpeech=NNP] [Text=for CharacterOffsetBegin=487 CharacterOffsetEnd=490 PartOfSpeech=IN] [Text=some CharacterOffsetBegin=491 CharacterOffsetEnd=495 PartOfSpeech=DT] [Text=nice CharacterOffsetBegin=496 CharacterOffsetEnd=500 PartOfSpeech=JJ] [Text=closeup CharacterOffsetBegin=501 CharacterOffsetEnd=508 PartOfSpeech=NN] [Text=shots CharacterOffsetBegin=509 CharacterOffsetEnd=514 PartOfSpeech=NNS] [Text=and CharacterOffsetBegin=515 CharacterOffsetEnd=518 PartOfSpeech=CC] [Text=some CharacterOffsetBegin=519 CharacterOffsetEnd=523 PartOfSpeech=DT] [Text=wonderful CharacterOffsetBegin=524 CharacterOffsetEnd=533 PartOfSpeech=JJ] [Text=color CharacterOffsetBegin=534 CharacterOffsetEnd=539 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=539 CharacterOffsetEnd=540 PartOfSpeech=,] [Text=but CharacterOffsetBegin=541 CharacterOffsetEnd=544 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=545 CharacterOffsetEnd=548 PartOfSpeech=DT] [Text=story CharacterOffsetBegin=549 CharacterOffsetEnd=554 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=555 CharacterOffsetEnd=557 PartOfSpeech=VBZ] [Text=so CharacterOffsetBegin=558 CharacterOffsetEnd=560 PartOfSpeech=RB] [Text=weak CharacterOffsetBegin=561 CharacterOffsetEnd=565 PartOfSpeech=JJ] [Text=- CharacterOffsetBegin=566 CharacterOffsetEnd=567 PartOfSpeech=:] [Text=no CharacterOffsetBegin=568 CharacterOffsetEnd=570 PartOfSpeech=DT] [Text=character CharacterOffsetBegin=571 CharacterOffsetEnd=580 PartOfSpeech=NN] [Text=development CharacterOffsetBegin=581 CharacterOffsetEnd=592 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=593 CharacterOffsetEnd=596 PartOfSpeech=CC] [Text=no CharacterOffsetBegin=597 CharacterOffsetEnd=599 PartOfSpeech=DT] [Text=plot CharacterOffsetBegin=600 CharacterOffsetEnd=604 PartOfSpeech=NN] [Text=- CharacterOffsetBegin=605 CharacterOffsetEnd=606 PartOfSpeech=:] [Text=it CharacterOffsetBegin=607 CharacterOffsetEnd=609 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=609 CharacterOffsetEnd=611 PartOfSpeech=VBZ] [Text=unable CharacterOffsetBegin=612 CharacterOffsetEnd=618 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=619 CharacterOffsetEnd=621 PartOfSpeech=TO] [Text=compensate CharacterOffsetBegin=622 CharacterOffsetEnd=632 PartOfSpeech=VB] [Text=. CharacterOffsetBegin=632 CharacterOffsetEnd=633 PartOfSpeech=.] 
root(ROOT-0, bad-6)
poss(smugness-3, Rourke-1)
possessive(Rourke-1, 's-2)
nsubj(bad-6, smugness-3)
cop(bad-6, is-4)
advmod(bad-6, so-5)
nsubj(sickening-9, it-7)
cop(sickening-9, 's-8)
ccomp(bad-6, sickening-9)
cc(sickening-9, and-10)
conj(sickening-9, Bassinger-11)
punct(bad-6, ,-12)
prep(bad-6, despite-13)
det(figure-16, the-14)
amod(figure-16, great-15)
pobj(despite-13, figure-16)
punct(bad-6, ,-17)
conj(bad-6, looks-18)
advmod(more-20, cheap-19)
acomp(looks-18, more-20)
prep(more-20, than-21)
pobj(than-21, beautiful.Kudos-22)
prep(looks-18, to-23)
det(photographer-25, the-24)
pobj(to-23, photographer-25)
prep(looks-18, for-26)
det(shots-30, some-27)
amod(shots-30, nice-28)
nn(shots-30, closeup-29)
pobj(for-26, shots-30)
cc(shots-30, and-31)
det(color-34, some-32)
amod(color-34, wonderful-33)
conj(shots-30, color-34)
punct(bad-6, ,-35)
cc(bad-6, but-36)
det(story-38, the-37)
nsubj(weak-41, story-38)
cop(weak-41, is-39)
advmod(weak-41, so-40)
conj(bad-6, weak-41)
punct(weak-41, --42)
det(development-45, no-43)
nn(development-45, character-44)
dep(weak-41, development-45)
cc(development-45, and-46)
det(plot-48, no-47)
conj(development-45, plot-48)
punct(unable-52, --49)
nsubj(unable-52, it-50)
cop(unable-52, 's-51)
dep(development-45, unable-52)
aux(compensate-54, to-53)
xcomp(unable-52, compensate-54)
punct(bad-6, .-55)

Sentence #6 (18 tokens):
Let's face it: this movie was made for only reason - to titillate male viewers.
[Text=Let CharacterOffsetBegin=638 CharacterOffsetEnd=641 PartOfSpeech=VB] [Text='s CharacterOffsetBegin=641 CharacterOffsetEnd=643 PartOfSpeech=PRP] [Text=face CharacterOffsetBegin=644 CharacterOffsetEnd=648 PartOfSpeech=VB] [Text=it CharacterOffsetBegin=649 CharacterOffsetEnd=651 PartOfSpeech=PRP] [Text=: CharacterOffsetBegin=651 CharacterOffsetEnd=652 PartOfSpeech=:] [Text=this CharacterOffsetBegin=653 CharacterOffsetEnd=657 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=658 CharacterOffsetEnd=663 PartOfSpeech=NN] [Text=was CharacterOffsetBegin=664 CharacterOffsetEnd=667 PartOfSpeech=VBD] [Text=made CharacterOffsetBegin=668 CharacterOffsetEnd=672 PartOfSpeech=VBN] [Text=for CharacterOffsetBegin=673 CharacterOffsetEnd=676 PartOfSpeech=IN] [Text=only CharacterOffsetBegin=677 CharacterOffsetEnd=681 PartOfSpeech=JJ] [Text=reason CharacterOffsetBegin=682 CharacterOffsetEnd=688 PartOfSpeech=NN] [Text=- CharacterOffsetBegin=689 CharacterOffsetEnd=690 PartOfSpeech=:] [Text=to CharacterOffsetBegin=691 CharacterOffsetEnd=693 PartOfSpeech=TO] [Text=titillate CharacterOffsetBegin=694 CharacterOffsetEnd=703 PartOfSpeech=VB] [Text=male CharacterOffsetBegin=704 CharacterOffsetEnd=708 PartOfSpeech=JJ] [Text=viewers CharacterOffsetBegin=709 CharacterOffsetEnd=716 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=716 CharacterOffsetEnd=717 PartOfSpeech=.] 
root(ROOT-0, Let-1)
nsubj(face-3, 's-2)
ccomp(Let-1, face-3)
dobj(face-3, it-4)
punct(Let-1, :-5)
det(movie-7, this-6)
nsubjpass(made-9, movie-7)
auxpass(made-9, was-8)
parataxis(Let-1, made-9)
prep(made-9, for-10)
amod(reason-12, only-11)
pobj(for-10, reason-12)
punct(reason-12, --13)
aux(titillate-15, to-14)
infmod(reason-12, titillate-15)
amod(viewers-17, male-16)
dobj(titillate-15, viewers-17)
punct(Let-1, .-18)

Sentence #7 (8 tokens):
On that level, it probably succeeded.
[Text=On CharacterOffsetBegin=722 CharacterOffsetEnd=724 PartOfSpeech=IN] [Text=that CharacterOffsetBegin=725 CharacterOffsetEnd=729 PartOfSpeech=DT] [Text=level CharacterOffsetBegin=730 CharacterOffsetEnd=735 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=735 CharacterOffsetEnd=736 PartOfSpeech=,] [Text=it CharacterOffsetBegin=737 CharacterOffsetEnd=739 PartOfSpeech=PRP] [Text=probably CharacterOffsetBegin=740 CharacterOffsetEnd=748 PartOfSpeech=RB] [Text=succeeded CharacterOffsetBegin=749 CharacterOffsetEnd=758 PartOfSpeech=VBD] [Text=. CharacterOffsetBegin=758 CharacterOffsetEnd=759 PartOfSpeech=.] 
root(ROOT-0, succeeded-7)
prep(succeeded-7, On-1)
det(level-3, that-2)
pobj(On-1, level-3)
punct(succeeded-7, ,-4)
nsubj(succeeded-7, it-5)
advmod(succeeded-7, probably-6)
punct(succeeded-7, .-8)

Sentence #8 (48 tokens):
If I recall, it's why I gave it a look being a fan of Bassinger's looks, but I actually expected a story, too.Those trying to pass this off as "arty" and something deeper than soft porn are only fooling themselves.
[Text=If CharacterOffsetBegin=764 CharacterOffsetEnd=766 PartOfSpeech=IN] [Text=I CharacterOffsetBegin=767 CharacterOffsetEnd=768 PartOfSpeech=PRP] [Text=recall CharacterOffsetBegin=769 CharacterOffsetEnd=775 PartOfSpeech=VBP] [Text=, CharacterOffsetBegin=775 CharacterOffsetEnd=776 PartOfSpeech=,] [Text=it CharacterOffsetBegin=777 CharacterOffsetEnd=779 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=779 CharacterOffsetEnd=781 PartOfSpeech=VBZ] [Text=why CharacterOffsetBegin=782 CharacterOffsetEnd=785 PartOfSpeech=WRB] [Text=I CharacterOffsetBegin=786 CharacterOffsetEnd=787 PartOfSpeech=PRP] [Text=gave CharacterOffsetBegin=788 CharacterOffsetEnd=792 PartOfSpeech=VBD] [Text=it CharacterOffsetBegin=793 CharacterOffsetEnd=795 PartOfSpeech=PRP] [Text=a CharacterOffsetBegin=796 CharacterOffsetEnd=797 PartOfSpeech=DT] [Text=look CharacterOffsetBegin=798 CharacterOffsetEnd=802 PartOfSpeech=NN] [Text=being CharacterOffsetBegin=803 CharacterOffsetEnd=808 PartOfSpeech=VBG] [Text=a CharacterOffsetBegin=809 CharacterOffsetEnd=810 PartOfSpeech=DT] [Text=fan CharacterOffsetBegin=811 CharacterOffsetEnd=814 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=815 CharacterOffsetEnd=817 PartOfSpeech=IN] [Text=Bassinger CharacterOffsetBegin=818 CharacterOffsetEnd=827 PartOfSpeech=NNP] [Text='s CharacterOffsetBegin=827 CharacterOffsetEnd=829 PartOfSpeech=POS] [Text=looks CharacterOffsetBegin=830 CharacterOffsetEnd=835 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=835 CharacterOffsetEnd=836 PartOfSpeech=,] [Text=but CharacterOffsetBegin=837 CharacterOffsetEnd=840 PartOfSpeech=CC] [Text=I CharacterOffsetBegin=841 CharacterOffsetEnd=842 PartOfSpeech=PRP] [Text=actually CharacterOffsetBegin=843 CharacterOffsetEnd=851 PartOfSpeech=RB] [Text=expected CharacterOffsetBegin=852 CharacterOffsetEnd=860 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=861 CharacterOffsetEnd=862 PartOfSpeech=DT] [Text=story CharacterOffsetBegin=863 CharacterOffsetEnd=868 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=868 CharacterOffsetEnd=869 PartOfSpeech=,] [Text=too.Those CharacterOffsetBegin=870 CharacterOffsetEnd=879 PartOfSpeech=NN] [Text=trying CharacterOffsetBegin=880 CharacterOffsetEnd=886 PartOfSpeech=VBG] [Text=to CharacterOffsetBegin=887 CharacterOffsetEnd=889 PartOfSpeech=TO] [Text=pass CharacterOffsetBegin=890 CharacterOffsetEnd=894 PartOfSpeech=VB] [Text=this CharacterOffsetBegin=895 CharacterOffsetEnd=899 PartOfSpeech=DT] [Text=off CharacterOffsetBegin=900 CharacterOffsetEnd=903 PartOfSpeech=RP] [Text=as CharacterOffsetBegin=904 CharacterOffsetEnd=906 PartOfSpeech=IN] [Text=`` CharacterOffsetBegin=907 CharacterOffsetEnd=908 PartOfSpeech=``] [Text=arty CharacterOffsetBegin=908 CharacterOffsetEnd=912 PartOfSpeech=JJ] [Text='' CharacterOffsetBegin=912 CharacterOffsetEnd=913 PartOfSpeech=''] [Text=and CharacterOffsetBegin=914 CharacterOffsetEnd=917 PartOfSpeech=CC] [Text=something CharacterOffsetBegin=918 CharacterOffsetEnd=927 PartOfSpeech=NN] [Text=deeper CharacterOffsetBegin=928 CharacterOffsetEnd=934 PartOfSpeech=JJR] [Text=than CharacterOffsetBegin=935 CharacterOffsetEnd=939 PartOfSpeech=IN] [Text=soft CharacterOffsetBegin=940 CharacterOffsetEnd=944 PartOfSpeech=JJ] [Text=porn CharacterOffsetBegin=945 CharacterOffsetEnd=949 PartOfSpeech=NN] [Text=are CharacterOffsetBegin=950 CharacterOffsetEnd=953 PartOfSpeech=VBP] [Text=only CharacterOffsetBegin=954 CharacterOffsetEnd=958 PartOfSpeech=RB] [Text=fooling CharacterOffsetBegin=959 CharacterOffsetEnd=966 PartOfSpeech=VBG] [Text=themselves CharacterOffsetBegin=967 CharacterOffsetEnd=977 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=977 CharacterOffsetEnd=978 PartOfSpeech=.] 
root(ROOT-0, 's-6)
mark(recall-3, If-1)
nsubj(recall-3, I-2)
advcl('s-6, recall-3)
punct('s-6, ,-4)
nsubj('s-6, it-5)
advmod(gave-9, why-7)
nsubj(gave-9, I-8)
advcl('s-6, gave-9)
iobj(gave-9, it-10)
det(look-12, a-11)
dobj(gave-9, look-12)
cop(fan-15, being-13)
det(fan-15, a-14)
xcomp(look-12, fan-15)
prep(fan-15, of-16)
poss(looks-19, Bassinger-17)
possessive(Bassinger-17, 's-18)
pobj(of-16, looks-19)
punct('s-6, ,-20)
cc('s-6, but-21)
nsubj(expected-24, I-22)
advmod(expected-24, actually-23)
conj('s-6, expected-24)
det(story-26, a-25)
dobj(expected-24, story-26)
punct(story-26, ,-27)
appos(story-26, too.Those-28)
partmod(too.Those-28, trying-29)
aux(pass-31, to-30)
xcomp(trying-29, pass-31)
dobj(pass-31, this-32)
prt(pass-31, off-33)
prep(pass-31, as-34)
punct(arty-36, ``-35)
pobj(as-34, arty-36)
punct(arty-36, ''-37)
cc(arty-36, and-38)
conj(arty-36, something-39)
amod(something-39, deeper-40)
mark(fooling-46, than-41)
amod(porn-43, soft-42)
nsubj(fooling-46, porn-43)
aux(fooling-46, are-44)
advmod(fooling-46, only-45)
ccomp(deeper-40, fooling-46)
dobj(fooling-46, themselves-47)
punct('s-6, .-48)

